lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2909,20 +2909,32 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(
       return exe_module_sp->GetArchitecture().GetTriple();
     };
 
+    ArchSpec active_arch;
```

Comment:
I think it would be less confusing if we called this `module_arch` or maybe `image_arch` (even though module is a very overloaded term).

Reply:
This is a variable that holds either the module/image arch, and if that's not available, holds the target arch. "Active" meaning, the one we found, in order of preference (module first, target second).

So it's not `module_arch`, but maybe `active_arch` is also not right.

Reply:
I'm going to rename it to `preferred_`… hopefully you'll agree that's ok.